tsl/src/bgw_policy/policies_v2.h

```diff
@@ -0,0 +1,105 @@
+/*
+ * This file and its contents are licensed under the Timescale License.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-TIMESCALE for a copy of the license.
+ */
+
+#include <postgres.h>
+#include <utils/jsonb.h>
+#include "dimension.h"
+#include <continuous_aggs/materialize.h>
+#include <bgw_policy/compression_api.h>
+#include <bgw_policy/continuous_aggregate_api.h>
+#include <bgw_policy/retention_api.h>
+
+#define POLICY_REFRESH_CAGG_PROC_NAME "policy_refresh_continuous_aggregate"
+#define POL_REFRESH_CONF_KEY_MAT_HYPERTABLE_ID "mat_hypertable_id"
+#define POL_REFRESH_CONF_KEY_START_OFFSET "start_offset"
+#define POL_REFRESH_CONF_KEY_END_OFFSET "end_offset"
+
+#define POLICY_COMPRESSION_PROC_NAME "policy_compression"
+#define POL_COMPRESSION_CONF_KEY_HYPERTABLE_ID "hypertable_id"
+#define POL_COMPRESSION_CONF_KEY_COMPRESS_AFTER "compress_after"
+#define POL_COMPRESSION_CONF_KEY_MAXCHUNKS_TO_COMPRESS "maxchunks_to_compress"
+
+#define POLICY_RECOMPRESSION_PROC_NAME "policy_recompression"
+#define POL_RECOMPRESSION_CONF_KEY_RECOMPRESS_AFTER "recompress_after"
+
+#define POLICY_RETENTION_PROC_NAME "policy_retention"
+#define POL_RETENTION_CONF_KEY_HYPERTABLE_ID "hypertable_id"
+#define POL_RETENTION_CONF_KEY_DROP_AFTER "drop_after"
+
+#define SHOW_POLICY_KEY_HYPERTABLE_ID "hypertable_id"
+#define SHOW_POLICY_KEY_POLICY_NAME "policy_name"
+#define SHOW_POLICY_KEY_REFRESH_INTERVAL "refresh_interval"
+#define SHOW_POLICY_KEY_REFRESH_START_OFFSET "refresh_start_offset"
+#define SHOW_POLICY_KEY_REFRESH_END_OFFSET "refresh_end_offset"
+#define SHOW_POLICY_KEY_COMPRESS_AFTER POL_COMPRESSION_CONF_KEY_COMPRESS_AFTER
+#define SHOW_POLICY_KEY_COMPRESS_INTERVAL "compress_interval"
+#define SHOW_POLICY_KEY_DROP_AFTER POL_RETENTION_CONF_KEY_DROP_AFTER
+#define SHOW_POLICY_KEY_RETENTION_INTERVAL "retention_interval"
+
+#define DEFAULT_RETENTION_SCHEDULE_INTERVAL                                                        \
+	{                                                                                              \
+		.day = 1                                                                                   \
+	}
+/*
+ * Default scheduled interval for compress jobs = default chunk length.
+ * If this is non-timestamp based hypertable, then default is 1 day
+ */
+#define DEFAULT_COMPRESSION_SCHEDULE_INTERVAL                                                      \
+	DatumGetIntervalP(DirectFunctionCall3(interval_in, CStringGetDatum("1 day"), InvalidOid, -1))
+
+#define DEFAULT_REFRESH_SCHEDULE_INTERVAL                                                          \
+	DatumGetIntervalP(DirectFunctionCall3(interval_in, CStringGetDatum("1 hour"), InvalidOid, -1))
```

Comment:
These makes sense to change names for since they are now global but the name did not reflect what policy it was default for.

Reply:
The prefix is either `_RETENTION_`, `_COMPRESSION_` or `_REFRESH_`, so I think they reflect the corresponding policies. I may be missing something.